apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-drive-explorer/AttachmentsDriveExplorerDrawer.vue

```diff
@@ -600,6 +600,7 @@ export default {
             
           } else {
             this.$root.$emit('attachments-default-folder-path-initialized', '/', this.currentDrive.title);
+            this.driveExplorerInitializing = false;
```

Comment:
We may need to change as well the code in https://github.com/exoplatform/ecms/blob/7b256459ed31f52a10c3b3cedc098ddee34cea94/apps/portlet-documents/src/main/webapp/vue-app/attachment/components/attachments-drive-explorer/AttachmentsDriveExplorerDrawer.vue#L578-L581

to make it as follow : 
```javascript
                this.openFolder(defaultFolder)
                  .then(() => {
                    this.$root.$emit('attachments-default-folder-path-initialized', this.getRelativePath(self.selectedFolderPath), this.schemaFolder);
                  }).finally(() => this.driveExplorerInitializing = false);
```

The `catch` case isn't handled by original code, thus either when the Promise gets resolved or rejected, in all cases, we need to stop loading UI effect.